src/ca65/pseudo.c

```diff
@@ -1043,6 +1043,12 @@ static void DoFeature (void)
             ErrorSkip ("Invalid feature: '%m%p'", &CurTok.SVal);
             return;
         }
+
+        if (Feature == FEAT_ADDRSIZE) {
+            /* Warn for depreciated .feature addrsize */
+            Warning (1, "Depreciated feature: '.feature addrsize'. Pseudo function .addrsize is always available.");
```

Comment:
typo, the right term is "deprecated". the comment above seems superfluous as the code is self-explanatory

Reply:
Thanks